packages/core/src/alert/alert-group.element.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('Alert groups – ', () => {
 
       layout = wrapper.getAttribute('cds-layout');
       expect(layout.includes('vertical')).toBe(true, 'Compact alert group should include vertical layout');
-      expect(layout.includes('no-wrap')).toBe(true, 'Compact alert group should include no-wrap layout');
+      expect(layout.includes('wrap:none')).toBe(true, 'Compact alert group should include wrap:none layout');
```

Comment:
Updated wrap to support params to support additional wrapping options (forms in next pr)